pages/comparison-tool.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <div>
+    <ComparisonToolHero />
+    <div class="container">
+      <div class="comparison_card-container mx-auto my-5">
+        <div
+          v-for="(selectedFB, index) in selectedFormBuildersOption"
+          :key="selectedFB + index"
+        >
+          <ComparisonOptionCard
+            :options="filteredOptions(index)"
+            :selected-option="selectedFB"
+            :card-number="index"
+            :form-builders-logo-src="formBuildersLogoSrc"
+            @change="handleOptionChange"
+          />
+        </div>
+      </div>
+      <FormBuilderComparisonTable
+        :selected-form-builders-details="selectedFormBuildersDetails"
+        :feature-name-list="featureNameList"
+        :form-builders="formBuilders"
+      />
+      <!-- <div
+        v-if="!selectedFormBuildersOption.filter(Boolean).length"
+        class="my-5"
+      >
+        <h2 class="section__heading">Popular Form Builder Comparisons</h2>
+        <div class="section__grid">
+          <ComparisonCard
+            imgSrc="/form-building-platforms/formester-vs-typeform.svg"
+            formBuilder1="Formester"
+            formBuilder2="Typeform"
+            @onViewComparisonButtonClick="handleViewComparisonClick"
+          />
+          <ComparisonCard
+            imgSrc="/form-building-platforms/jotform-vs-typeform.svg"
+            formBuilder1="Jotform"
+            formBuilder2="Typeform"
+            @onViewComparisonButtonClick="handleViewComparisonClick"
+          />
+          <ComparisonCard
+            imgSrc="/form-building-platforms/formester-vs-jotform.svg"
+            formBuilder1="Formester"
+            formBuilder2="Jotform"
+            @onViewComparisonButtonClick="handleViewComparisonClick"
+          />
+        </div>
+      </div> -->
```

Comment:
Why is this commented?

Reply:
we will be using this once we have right information are updated for the comparison tool through the cms